geniza/corpus/metadata_export.py

```diff
@@ -0,0 +1,219 @@
+CSV_EXPORT_PROGRESS = True
+SEP_WITHIN_CELLS = "; "
+CSV_EXPORT_FIELDS_DOC = [
+    "pgpid",
+    "url",
+    "iiif_urls",
+    "fragment_urls",
+    "shelfmark",
+    "multifragment",
+    "side",
+    "region",
+    "type",
+    "tags",
+    "description",
+    "shelfmarks_historic",
+    "languages_primary",
+    "languages_secondary",
+    "language_note",
+    "doc_date_original",
+    "doc_date_calendar",
+    "doc_date_standard",
+    "notes",
+    "needs_review",
+    "url_admin",
+    "initial_entry",
+    "last_modified",
+    "input_by",
+    "status",
+    "library",
+    "collection",
+]
+
+import csv
+import os
+
+from django.conf import settings
+from django.contrib.postgres.aggregates import ArrayAgg
+from django.contrib.sites.models import Site
+from django.db.models.query import Prefetch
+from django.http import StreamingHttpResponse
+from django.utils import timezone
+from rich.progress import track
+
+script_user = settings.SCRIPT_USERNAME
+site_domain = Site.objects.get_current().domain.rstrip("/")
+url_scheme = "https://"
+
+from geniza.common.utils import Echo, timeprint
+from geniza.corpus.models import Document, TextBlock
+
+
+class Exporter(object):
```

Comment:
I like the exporter class structure! It seems like this should work pretty well to adapt to the other exports we need to implement. 

FYI, in python 3 you can omit the `(object)` — this is exactly equivalent:

```suggestion
class Exporter:
```